CONTRIBUTING.md

```diff
@@ -3,7 +3,7 @@
 You are welcome to contribute to this project. Here are the guidelines we try to stick to in this project.
 
 
-* [Question or Problem](#question-or-problem)
+* [Questions or Problems](#question-or-problem)
```

Comment:
Can you update the link? GitHub automatically makes an anchor based on the heading, so it should be `(#questions-or-problems)`.